modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -109,13 +111,13 @@ CREATE TABLE IDN_OAUTH2_ACCESS_TOKEN_SCOPE (
 
 IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[IDN_OAUTH2_SCOPE]') AND TYPE IN (N'U'))
 CREATE TABLE IDN_OAUTH2_SCOPE (
-            SCOPE_ID INTEGER IDENTITY,
-            SCOPE_KEY VARCHAR(100) NOT NULL,
-            NAME VARCHAR(255) NULL,
-            DESCRIPTION VARCHAR(512) NULL,
-            TENANT_ID INTEGER NOT NULL DEFAULT 0,
-            ROLES VARCHAR (500) NULL,
-            PRIMARY KEY (SCOPE_ID)
+  			SCOPE_ID INTEGER IDENTITY,
+  			SCOPE_KEY VARCHAR(100) NOT NULL,
+  			NAME VARCHAR(255) NULL,
+  			DESCRIPTION VARCHAR(512) NULL,
+  			TENANT_ID INTEGER NOT NULL DEFAULT 0,
+			ROLES VARCHAR (500) NULL,
+  			PRIMARY KEY (SCOPE_ID)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04